deploy/Makefile

```diff
@@ -1,21 +1,50 @@
 ENV_FILE=.env
 include $(ENV_FILE)
 
-DOCKER_REMOTE=ssh://$(ZEN_WEB_REMOTE)
-PROJECT_NAME=harmony
+DOCKER_HOST ?= ssh://$(ZEN_WEB_REMOTE)
+PROJECT_NAME ?= harmony-web
 
-web_configure:
+configure:
 	@scp nginx/nginx_vhost_default_location $(ZEN_WEB_REMOTE):${NGINX_DEFAULT_VHOST_CONFIG}
 	@scp instance_config.json $(ZEN_WEB_REMOTE):${INSTANCE_CONFIG}
 	@scp global_config.py $(ZEN_WEB_REMOTE):${GLOBAL_CONFIG}
 
-web_deploy:
-	@DOCKER_HOST=$(DOCKER_REMOTE) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml up -d --force-recreate
-	@DOCKER_HOST=$(DOCKER_REMOTE) docker system prune -af
+prune:
+	@DOCKER_HOST=$(DOCKER_HOST) docker system prune -af
 
-web_stop:
-	@DOCKER_HOST=$(DOCKER_REMOTE) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml down
-	@DOCKER_HOST=$(DOCKER_REMOTE) docker system prune -af
+up:
```

Comment:
I changed the convention - used to be "web_deploy" and "web_stop" - which got confusing when I want to stop the web container - so now you do `make up` where you used to do `make web_deploy`

Reply:
mmm don't see the confusion here, but I see you have a bunch of other commands so okay.

Reply:
yep - it's the clash with the other commands. the moment I wanted to have a convenience function for a specific container, the commands as named before just didn't make sense anymore.